IndicatorFields/incidentfield-threattypes_CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
+Field changed from text to grid.
```

Comment:
```suggestion
```

Reply:
We should remove this. Since this indicator field was already `"fromVersion": "5.5.0"`, this hasn't gone out to customers, so since it's a change for something that hasn't technically been released you should remove the release notes line.